src/composed/signed_key/mod.rs

```diff
@@ -87,6 +88,7 @@
 //!         packet::Subpacket::regular(packet::SubpacketData::Issuer(signing_key.key_id())),
 //!     ],
 //!     vec![],
+//!     SignatureVersionSpecific::V4,
```

Comment:
can we instead have `Signature::v4`?

Reply:
So, to clarify: you're thinking of adding `::vX()` constructors to the `SignatureVersionSpecific` enum?

We're not constructing that type a lot, anymore, I believe.

Reply:
no, I mean adding a `vX` constructor to `Signature`, replacing the generic `new` constructor

Reply:
right, that looks good 3750727b0684f7c64b6cec086a1cb028b205d5aa